### PR TITLE
Failed flushes alert

### PR DESCRIPTION
**Pull request by joe-elliott**
**What this PR does**:
Makes the `TempoIngesterFlushesFailing` alert less trigger-happy in situations where it can self-heal.
It also includes a variation on the alert which causes it to page immediately if self-healing is failing.

**Which issue(s) this PR fixes**:
This completes the follow-up I should have done after https://github.com/grafana/tempo/pull/1049

**Checklist**
- [ ] Tests updated
- [ ] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: operations/tempo-mixin/yamls/alerts.yaml [hunk 1]
```diff
@@ -51,13 +51,24 @@
...
     "labels":
       "severity": "critical"
+  - "alert": "TempoIngesterFlushesFailing"
+    "annotations":
+      "message": "Greater than 2 flushes have failed in the past hour, and retries have failed."
+      "runbook_url": "https://github.com/grafana/tempo/tree/main/operations/tempo-mixin/runbook.md#TempoIngesterFlushesFailing"
+    "expr": |
+      sum by (cluster, namespace) (increase(tempo_ingester_failed_flushes_total{}[1h])) > 2 and
+      sum by (cluster, namespace) (increase(tempo_ingester_failed_flushes_total{}[5m])) > 0 and
+      sum by (cluster, namespace) (increase(tempo_ingester_flush_failed_retries_total{}[5m])) > 0
```
Comment (dannykopping) [inline operations/tempo-mixin/yamls/alerts.yaml:61]: I think this file needs to be regenerated @joe-elliott

**Review by dannykopping — APPROVED** [review]
:+1:

File: operations/tempo-mixin/yamls/alerts.yaml (hunk 1 above)
Comment (joe-elliott) [reply operations/tempo-mixin/yamls/alerts.yaml:61]: yup, thx!

Comment (kvrhdn) [reply operations/tempo-mixin/yamls/alerts.yaml:61]: +1. Normally CI should fail on this, but it seems that GitHub Actions didn't trigger?

**Review by kvrhdn — APPROVED** [review]
LGTM :+1:

**Pull request merged**